### PR TITLE
Make configurable cubestore metrics address

### PR DESCRIPTION
**Pull request by fbalicchia**
Hi,
could you please review this pr. The capability that tried to add is the following describe in this https://github.com/cube-js/cube.js/issues/6314

### CODE REVIEW

File: rust/cubestore/cubestore/src/bin/cubestored.rs [hunk 1]
```diff
@@ -22,16 +22,21 @@ fn main() {
...
         ),
         Err(_) => metrics::Compatibility::StatsD,
     };
-    init_metrics("127.0.0.1:0", "127.0.0.1:8125", metrics_mode, vec![]);
+    let metrics_addr =
+        std::env::var("CUBESTORE_METRICS_ADDRESS").unwrap_or("127.0.0.1".to_string());
+    let metrics_port = std::env::var("CUBESTORE_METRICS_PORT").unwrap_or("8125".to_string());
+    let metrics_server_address = format!("{}:{}", metrics_addr, metrics_port);
+
+    init_metrics("127.0.0.1:0", metrics_server_address, metrics_format);
```
Comment (ovr) [inline rust/cubestore/cubestore/src/bin/cubestored.rs:39]:
```suggestion
    init_metrics("127.0.0.1:0", metrics_server_address, metrics_format, vec![]);
```

Comment (fbalicchia) [reply rust/cubestore/cubestore/src/bin/cubestored.rs]: thanks, @ovr I've done i blind merge now I've fixed it. build failed again I tried to understand

**Comment by fbalicchia** [comment]
Hi, @ovr pr seems ready when you have a couple of minutes could you please review 
 Thanks

File: docs/content/Reference/Configuration/Environment-Variables-Reference.mdx [hunk 2]
```diff
@@ -887,6 +887,31 @@ If `true`, then send telemetry to Cube.
...
+Define which metrics collector format.
+
+| Possible Values | Default in Development | Default in Production |
+| --------------- | ---------------------- | --------------------- |
+| `statsd`, `dogstatsd` | `statsd`                 | `statsd`                |
+
+
+## `CUBESTORE_METRICS_ADDRESS`
+
+Required ipAddress to send metrics.
```
Comment (hassankhan) [inline docs/content/Reference/Configuration/Environment-Variables-Reference.mdx:901]:
```suggestion
Required IP address to send metrics.
```

File: docs/content/Reference/Configuration/Environment-Variables-Reference.mdx [hunk 3]
```diff
@@ -887,6 +887,31 @@ If `true`, then send telemetry to Cube.
...
+
+| Possible Values | Default in Development | Default in Production |
+| --------------- | ---------------------- | --------------------- |
+| A valid ip address | `127.0.0.1`                 | `127.0.0.1`                |
```
Comment (hassankhan) [inline docs/content/Reference/Configuration/Environment-Variables-Reference.mdx:905]:
```suggestion
| A valid IP address | `127.0.0.1`                 | `127.0.0.1`                |
```

File: docs/content/Reference/Configuration/Environment-Variables-Reference.mdx [hunk 4]
```diff
@@ -887,6 +887,31 @@ If `true`, then send telemetry to Cube.
...
+
+
+## `CUBESTORE_METRICS_PORT`
+
+Required port to send where collector server accept UDP connections .
```
Comment (hassankhan) [inline docs/content/Reference/Configuration/Environment-Variables-Reference.mdx:910]:
```suggestion
Required port to send where collector server accept UDP connections.
```

File: docs/content/Reference/Configuration/Environment-Variables-Reference.mdx [hunk 5]
```diff
@@ -887,6 +887,31 @@ If `true`, then send telemetry to Cube.
...
+
+| Possible Values | Default in Development | Default in Production |
+| --------------- | ---------------------- | --------------------- |
+| A valid port number | `8125`                 | `8125`                |
```
Comment (hassankhan) [inline docs/content/Reference/Configuration/Environment-Variables-Reference.mdx:914]:
```suggestion
| A valid port number | `8125`                 | `8125`                |

```

**Review by hassankhan — CHANGES_REQUESTED** [review]
Minor nits, thanks for contributing @fbalicchia 💯

**Comment by fbalicchia** [comment]
@hassankhan thanks for the review. In the documentation, I found the following command to [format](https://github.com/cube-js/cube/blob/master/docs/README.md#formatting) docs. Do I need to run too?
I ask this cause I've run in dry-run mode but it change different things

**Comment by hassankhan** [comment]
@fbalicchia Don't worry about the formatting 😀

**Pull request merged**